src/App.tsx

```diff
@@ -1,14 +1,70 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Todo } from './types/Todo';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions as todoActions } from './features/todos';
+
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const allTodo = useAppSelector(state => state.todos);
+  const setAllTodo = (todos: Todo[]) => dispatch(todoActions.add(todos));
+  const formItems = useAppSelector(state => state.filter);
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [filterTodo, setFilterTodo] = useState(allTodo);
+  const [visibleTodo, setVisibleTodo] = useState(filterTodo);
+  const [selectUserId, setSelectUserId] = useState(0);
+  const [selectTodoId, setSelectTodoId] = useState(0);
+  const selectItems = (UserId: number, todoId: number) => {
+    setSelectUserId(UserId);
+    setSelectTodoId(todoId);
+  };
+
+  const handleSelect = () => {
+    switch (formItems.statusSelect) {
+      case 'active':
+        setFilterTodo(allTodo.filter(todo => !todo.completed));
+        break;
+
+      case 'completed':
```

Comment:
Active and Completed are magic words.
Consider will be better if you create a enum for magic words and use it everywhere.